app/src/main/res/layout/fragment_habit_entry.xml

```diff
@@ -27,6 +26,14 @@
             android:autofillHints="@string/habit_title"
             />
 
+        <Switch
+            android:id="@+id/edit_publicity_switch"
+            style="@style/TextStyle_Heading2"
+            android:layout_width="143dp"
+            android:layout_height="wrap_content"
+            android:fontFamily="@font/work_sans_bold"
+            android:text="Private" />
```

Comment:
The label here differs from the label given in habit details (this says private while the other one says public). We probably want to stay consistent